100-Resources/200-Contributing/200-Contribute-to-Docs.mdx

```diff
@@ -43,13 +44,35 @@ To ensure your pull request passes checks, apply the linter by navigating to the
 
 ### Troubleshoot linting
 
-To fix issues from linting, locate the exact error message that references the specific number. Examine the code or documentation related to the error and ensure the syntax and usage align with coding and documentation standards.
+To fix issues from linting, locate the exact error message that references the specific line number of tf the error. Examine the code or documentation related to the error and ensure the syntax and usage align with coding and documentation standards.
 If the error persists, contact us in [Discord](https://xata.io/discord) or use community resources for guidance.
 
+After adding and committing code changes, the linter may identify errors. For instance, in the following example, an error is found on **line 62**, along with four additional warnings.
+
+![Linting errors](images/linting-error.png)
+
+To resolve the issue, go to **line 62** in your code, correct the error, then add, commit, and push the changes to create a pull request.
+
```

Comment:
Since this is no longer mandatory, I'd remove anything related to linter. The bot will fix them and we don't scare new contributors

Reply:
Since we mention this in the docs repo README and some of the checks have failed, I think we should leave this in for now.

Reply:
Some checks might always fail for outsider contributors (those that require secrets like preview build), and in practice the reviewer will make sure they pass before merging. IMO we should make it easy for anyone to contribute and hide as much complexity as we can.